### PR TITLE
Review tile setup

### PR DESCRIPTION
**Pull request by tchitrakorn**
- Basic setup for ReviewTile component in Ratings and Reviews included
- CSS file added in the public folder
- Ratings and Reviews routing removed from the server-side (transitioning to wildcard method)
- Note: FontAwesome (for star ratings feature) is currently being added directly in index.html. May not be the best approach and require some modification if it will actually be used

### CODE REVIEW

**Comment by tchitrakorn** [comment]
Hi all, I have pushed my current work for Ratings and Reviews along with the CSS file. There are a lot of pieces in this PR, so feel free to let me know if anything should be adjusted. Thank you! @matthewwrobel @jabrake @alizehrehman

File: client/ratings_reviews/components/ReviewTile.jsx [hunk 1]
```diff
@@ -1,9 +1,42 @@
...
+  }
+
+  render() {
+    var responseVisibility = true;
+    if (this.props.review.response === null || this.props.review.response.length === 0) {
+      responseVisibility = false;
+    }
+    return (
+      <div class='review-tile'>
+        <div id={this.props.review.review_id}>Rating: {this.props.review.rating}
```
Comment (jabrake) [inline client/ratings_reviews/components/ReviewTile.jsx:26]: Curious about the choice of id for the div here, would you use that to target via CSS rules?

File: public/index.html [hunk 2]
```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="utf-8">
     <title>Project Atelier</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (jabrake) [inline public/index.html:7]: Is this a font library?

Comment (jabrake) [reply public/index.html:7]: Oh never mind I think I see now - it's to display the little star icons?

**Review by jabrake — APPROVED** [review]
This looks overall pretty good to me - it's just showing there's some merge conflicts in the server/index.js file. If we can clear that up I think we should be good to merge, unless anyone else wants to give this a quick look.

**Pull request closed without merging**